### PR TITLE
Aep

### PR DESCRIPTION
**Pull request by KamillaKhabibrakhmanova**

### CODE REVIEW

File: lib/facebook.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const FB_EVENT_FIELDS_STRING = [
 FB.setVersion(process.env.FB_GRAPH_API_VERSION || '2.8');
 FB.setAccessToken(process.env.FB_GRAPH_API_TOKEN);
 
+
```
Comment (aaghevli) [inline lib/facebook.js]: newline?

File: lib/links.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const config = require('../config.js');
+
+const apiBaseUrl = config.baseUrl + '/v1/';
+
+const links = {
+	self: {
```
Comment (aaghevli) [inline lib/links.js]: spacing conflicts with the linter (2 spaces per tab)

File: routes/handlers/entryPoint.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const osdiLinks = require('../../lib/links.js');
+
+exports.get = {
+
+  handler: function (req, reply) {
+    reply({
+      motd: 'Welcome to the Resistance Calendar API Entry Point',
+      max_pagesize: 25,
```
Comment (aaghevli) [inline routes/handlers/entryPoint.js]: Can we add this number to the config and use also in the [events.js#7](https://github.com/ResistanceCalendar/resistance-calendar-api/blob/master/routes/handlers/event.js#L7)

File: routes/handlers/entryPoint.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      vendor_name: 'Resistance Calendar',
+      product_name: 'Resistance Calendar OSDI Server',
+      osdi_version: '1.0',
+      namespace: 'resistance_calendar_api',
```
Comment (aaghevli) [inline routes/handlers/entryPoint.js]: resistance_calendar?

**Comment by KamillaKhabibrakhmanova** [comment]
@aaghevli ran the linter through and updated the page size to be set in config

**Comment by aaghevli** [comment]
Thanks @KamillaKhabibrakhmanova !! Getting close to official now!

**Pull request merged**